src/components/test/common-tests.js

```diff
@@ -334,7 +334,10 @@ export function testTransitionComponent(
           onTransitionEnd,
         });
 
-        wrapper.find('div').prop('ontransitionend')();
+        wrapper.find('div').prop('ontransitionend')({
+          target: wrapper.find('div').getDOMNode(),
+          ...event,
+        });
```

Comment:
This is a little hard to human-parse. Can you help me understand the change here?

Reply:
Basically, `ontransitionend` now expects an event object to be passed, and will return unless the target is the wrapper's first children, and the `propertyName` is `'height'`.

First I thought I could just pass the event here, as in `wrapper.find('div').prop('ontransitionend')(event)`, but determining the target from the outside of the function call requires knowing this function implementation. That's why I `target`'s default value is `wrapper.find('div').getDOMNode()`, but you can still override it.

I could perhaps document this better with a/some comment.